samples/bluetooth/mesh/light_ctrl/prj.conf

```diff
@@ -52,10 +52,11 @@ CONFIG_BT_MESH_DK_PROV=y
 CONFIG_BT_MESH_NLC_PERF_CONF=y
 CONFIG_BT_MESH_NLC_PERF_LIGHTNESS_CTRL=y
 CONFIG_BT_MESH_MODEL_EXTENSIONS=y
-CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE=18
+CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE=19
 # Enabling BT_MESH_NLC_PERF_CONF enables support for 3 application keys by
 # default. Therefore, allow up to 3 application key bindings per model instance.
 CONFIG_BT_MESH_MODEL_KEY_COUNT=3
+CONFIG_BT_MESH_LARGE_COMP_DATA_SRV=y
```

Comment:
I would recommend to revert these changes.

Reply:
It is in a same way like Health Server. Where we don't add metadata for health server in every sample. That is fully tested via tests. We should perhaps do the same and keep this sample compact and not add non-required features here. This is our showcase sample for BLCNLCP + ENMNLCP profiles and they do not mandate metadata.

Reply:
Then we do not need to add metadata in sample